chef_master/source/install_chef_air_gap.rst

```diff
@@ -0,0 +1,602 @@
+
+After setup successfully completes and a configuration has been applied, login credentials are reported in the completion output; however, they are also saved to ``/etc/delivery/ENTERPRISE_NAME-admin-credentials``.
+
+If you plan on using the workflow capabilities of Automate, proceed to the next section to setup your build nodes/runners. After they are set up, you can attempt to run an initial application or cookbook change through your Chef Automate server.
+
+For more information about ``automate-ctl`` and how to use it, see :doc:`ctl_automate_server`.
+
+Configure node data collection
+------------------------------------------------------------
+
+Configure a Data Collector token in Chef Automate
++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+All messages sent to Chef Automate are performed over HTTP and are authenticated with a pre-shared key called a ``token.`` Every Chef Automate installation configures a default token by default, but we strongly recommend that you create your own.
+
+To set your own token, add the following to your ``/etc/delivery/delivery.rb`` file:
+
+.. code-block:: ruby
+
+   data_collector['token'] = 'sometokenvalue'
+
+... and then run ``automate-ctl reconfigure``
+
+If you do not configure a token, the default token value is: ``93a49a4f2482c64126f7b6015e6b0f30284287ee4054ff8807fb63d9cbd1c506``
+
+
+Configure your Chef server to point to Chef Automate
++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+In addition to forwarding Chef run data to Automate, Chef server will send messages to Chef Automate whenever an action is taken on a Chef server object, such as when a cookbook is uploaded to the Chef server or when a user edits a role.
+
+To enable this feature on Chef Server versions 12.14 and later, channel the token setting through our veil secrets library because the token is considered a secret and, as such, cannot appear in ``/etc/opscode/chef-server.rb``. On Chef Server versions 12.14 and above, you must make the following to change the data collector token:
+
+.. code-block:: ruby
+
+   chef-server-ctl set-secret data_collector token 'TOKEN'
+   chef-server-ctl restart nginx
+
+To enable this feature on Chef Server versions 12.13 and earlier, add the following settings to ``/etc/opscode/chef-server.rb`` on the Chef server:
+
+.. code-block:: ruby
+
+   data_collector['root_url'] = 'https://my-automate-server.mycompany.com/data-collector/v0/'
+   data_collector['token'] = 'TOKEN'
+
+where ``my-automate-server.mycompany.com`` is the fully-qualified domain name of your Chef Automate server, and
+``TOKEN`` is either the default value or the token value you configured in the `prior section <#configure-a-data-collector-token-in-chef-automate>`__.
+
+Save the file and run ``chef-server-ctl reconfigure`` to complete the process.
+
+Additional configuration options include:
```

Comment:
We may want to mention `opscode_erchef['max_request_size']` as a tunable.  Its useful when proxying audit runs of large inspec profiles via the chef-server.

Reply:
Thanks for all your help with this, @galenemery!